rfq/manager.go

```diff
@@ -264,12 +267,18 @@ func (m *Manager) startSubsystems(ctx context.Context) error {
 		SpecifierChecker:  m.AssetMatchesSpecifier,
 		NoOpHTLCs:         m.cfg.NoOpHTLCs,
 		AuxChanNegotiator: m.cfg.AuxChanNegotiator,
+		PolicyStore:       m.cfg.PolicyStore,
 	})
 	if err != nil {
 		return fmt.Errorf("error initializing RFQ order handler: %w",
 			err)
 	}
 
+	if err := m.restorePersistedPolicies(ctx); err != nil {
+		return fmt.Errorf("error restoring persisted RFQ "+
+			"policies: %w", err)
+	}
```

Comment:
I think this should happen in `m.orderHandler.Start`. Effectively, I think `PolicyStore` should only be accessible by `OrderHandler`.